mailpoet/assets/js/src/wizard/steps/sender_step.tsx

```diff
@@ -52,47 +72,32 @@ function WelcomeWizardSenderStep(props) {
               name="senderAddress"
               type="text"
               placeholder="john@doe.com"
-              value={props.sender ? props.sender.address : ''}
+              value={sender ? sender.address : ''}
```

Comment:
You can replace this with [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) as well

Reply:
That is good to know. I don't think it is necessary to create a commit to change it in this case, but I will try to remember next time. Let me know if you disagree and you think I should change it in this PR.

Reply:
It's fine to leave it like this.